src/layout/FooterTraditional.js

```diff
@@ -72,6 +72,12 @@ const FooterTraditional = () => {
                     All Policies and Guidelines
                   </a>
                 </li>
+                <li>
+                  <a
+                    id="teconsent"
+                    className="nav-link land-p-footer__cookies"
+                  ></a>
```

Comment:
❓ missing label and onClick handler/href or am I missing something?

Reply:
Nope, the content and oncClick binding is injected by the cookies script in chrome.